app/rest/platformroutes.ts

```diff
@@ -180,4 +180,54 @@ export async function platformroutes(
 			});
 		});
 	});
+
+	/**
+	 * *
+	 * Block by block number
+	 * GET /fetchDataByBlockNo
+	 * curl -i 'http://<host>:<port>/fetchDataByBlockNo/<channel_genesis_hash>/<blockNo>'
+	 */
+	router.get('/fetchDataByBlockNo/:channel_genesis_hash/:blockNo', (req, res) => {
+		const blockNo = req.params.blockNo;
+		const channel_genesis_hash = req.params.channel_genesis_hash;
+		proxy.fetchDataByBlockNo(req.network, channel_genesis_hash, blockNo).then((data: any) => {
```

Comment:
[nitpick]: Keep the style consistent async await/promise.

Reply:
Hi @adityajoshi12 , Followed the same coding styles which used in other routes and functions, can you brief what change you are expecting?
Thanks !

Reply:
I see usage of  both `async-await` and `promise`, we can have a consistency across the project, maybe it could be handled in separate PR as well.

Reply:
Its been an observation from my side too
Since it is spread across all the code, I think its better to handle code consistency in another PR